ansible/roles/create-s390x-vm/tasks/main.yml

```diff
@@ -0,0 +1,74 @@
+- name: Fetch the variables from var file
+  include_vars:
+    file: vars.yml
+
+- name: Check for existing VSI
+  ibm.cloudcollection.ibm_is_instance_info:
+    name: "{{ vsi_name }}"
+  failed_when:
+    - vsi.rc != 0
+    - '"No Instance found" not in vsi.stderr'
+  register: vsi
+
+- name: Configure VSI
+  ibm.cloudcollection.ibm_is_instance:
+    name: "{{ vsi_name }}"
+    state: available
+    id: "{{ vsi.resource.id | default(omit) }}"
+    vpc: "{{ vpc_id }}"
```

Comment:
Any way we could create this id dynamically to spin up VMs on demand? Or are we stuck with the single VM? Our integration tests expect to be running on their own, so having a single VM in an environment where multiple CI runs could be triggered in parallel will be a problem, if we are to use a single VM, have you put some thought on how we could coordinate multiple CI jobs?

Reply:
The vm creation is solely determined by `vsi_name`, equivalent to `vm_name` for x86 vms (https://github.com/stackrox/collector/blob/master/ansible/roles/create-vm/tasks/main.yml#L35-L38). If x86 is creating these ID dynamically during the CI process, then it should be the same for the z vms.